demisto_sdk/commands/common/constants.py

```diff
@@ -503,6 +503,7 @@ class FileType(str, Enum):
     "jliiBUXY1lWWs9Hs+PgZlCMAXNBIRoRFMH1JDZzgeyl5/MFthk+FY4FyyyEqEzxKbI2DQxMv7MMJnQZDvovVqz"
     "A1HUzK9kdzt0/2+exnQr4g2hrAAAAABJRU5ErkJggg=="
 )
+DOCKER_IMAGE_REGEX = r"^demisto/([^\s:]+):(\d+(\.\d+)*)$"
```

Comment:
in this case how can i build a regex for external repos which i do not have the option to know their names?
what are the criterias that the regex would be based upon?

Reply:
it can be anything. why need a specific match?
If they give me 
asdasdasdad
then we will try to run 
```
docker inspect asdasdasdad
docker pull asdasdasdad
```
etc